apis/flinkcluster/v1beta1/flinkcluster_types_util.go

```diff
@@ -9,6 +9,12 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+const (
+	HA_CONFIG_HA_TYPE        = "high-availability"
+	HA_CONFIG_HA_STORAGE_DIR = "high-availability.storageDir"
+	HA_CONFIG_K8S_CLUSTER_ID = "kubernetes.cluster-id"
```

Comment:
This is not very idiomatic; let's have it MixedCaps; since we are not exporting it like `haConfigType`